meta/saisanitycheck.c

```diff
@@ -332,6 +332,26 @@ bool sai_metadata_is_acl_field_or_action(
     return false;
 }
 
+bool sai_metadata_is_tlv_or_segment(
```

Comment:
Structss but as list like s32list etc add default value empty And modify saisanitycheck adding those new 2 values to accepting empty as default, there is a section there on chek